tests/utils/whitelist.test.js

```diff
@@ -32,6 +32,13 @@ describe('validate url with whitelist', function() {
             return expect(checkWhitelist(url, whitelist)).toBeTruthy();
         });
 
+        it('should return true for non-standard TLDs', function() {
```

Comment:
also, please write a negative test case for this scenario.

Reply:
@lneir for example if the whitelist is:
```javascript 
[ "https://gs.symphony.com", "https://gs-uat.symphony.com" ]
```
users can visit -> 
`https://x.gs.symphony.com`,`https://y.gs-uat.symphony.com` and not `symphony.com`

Whereas if the whitelist is:
```javascript
[ 'www.symphony.com, app.symphony.com, symphony.com' ]
```

users can visit -> 
`https://xyz.symphony.com/`,  `https://abc.symphony.com/` and `symphony.com`

Reply:
ok just checking if you allow asterisk.  appears not.  the asterisk is implicit.